.circleci/real_config.yml

```diff
@@ -2023,6 +2030,7 @@ jobs:
           PW_SERVER_ADDRESS=${PW_SERVER_ADDRESS} \
           DET_WEBPACK_PROXY_URL=${DET_WEBPACK_PROXY_URL} \
           DET_WEBPACK_PROXY_URL=${DET_WEBSOCKET_PROXY_URL} \
+          PW_EE=<< parameters.ee >> \
```

Comment:
this is getting long and unwieldy. can we use bashrc instead?

Reply:
oh, use this
https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-step

Reply:
I will try it. I suspect there's shenanigans with npx/npm starting things in another process which is why we are feeding environment variables in-line like this.

Reply:
just deleting these was fine. They are apparently unneeded.